jsk_pcl_ros_utils/src/pointcloud_to_pcd_nodelet.cpp

```diff
@@ -112,16 +112,16 @@ namespace jsk_pcl_ros_utils
     {
       if(compressed_)
       {
-        ROS_INFO_STREAM ("Saving as binary compressed PCD");
+        NODELET_INFO_STREAM ("Saving as binary compressed PCD");
       }
       else
       {
-        ROS_INFO_STREAM ("Saving as binary PCD");
+        NODELET_INFO_STREAM ("Saving as binary PCD");
       }
     }
     else
     {
-      ROS_INFO_STREAM ("Saving as binary PCD");
+      NODELET_INFO_STREAM ("Saving as binary PCD");
```

Comment:
Could you please fix  the content of the info as well? 
Probably mistaken for ASCII format.

Reply:
Do you mean dec0d0e4b6012e8300834635c960a040086911f8 ?